src/draw.c

```diff
@@ -347,31 +347,34 @@ static GSList *create_layouts(cairo_t *c)
 {
         GSList *layouts = NULL;
 
-        int qlen = queues_length_waiting();
-        bool xmore_is_needed = qlen > 0 && settings.indicate_hidden;
+        int waiting_len = queues_length_waiting();
+        bool xmore_is_needed = waiting_len > 0 && settings.indicate_hidden;
 
         notification *last = NULL;
+        int slots_avail = settings.geometry.h;
```

Comment:
What's the point of this variable? If there are notifications in `waiting` we definitely want to have an `xmore`

Reply:
I guess, based on the current logic, I can remove it again. But while writing the initial code, I had a constraint and it was necessary to check it.